cloudman/projman/kubectl/client.py

```diff
@@ -0,0 +1,59 @@
+"""A wrapper around the kubectl commandline client"""
+import shutil
+from . import helpers
+
+
+class KubeCtlService(object):
```

Comment:
Would probably make sense to move the kubectl client to helmsman, since projman depends on helmsman and helmsman is fairly standalone. helmsman should probably retain the ability to create namespaces independent of projman.

Reply:
When you say _retain the ability to create namespaces_, do you mean in addition to projman? One use case that comes to mind is the AnVIL where Helmsman will be given a namespace to run in and won't be able to create its own namespaces so we should have allow the namespace to be optional/supplied.

Reply:
I was thinking of Anvil, and thought that perhaps it would be necessary to create the namespace before installing. In any case, helmsman should probably continue to retain the ability to create namespaces, even if optionally.

Reply:
I added the optional flag `--create_namespace` to `add_chart` command, so we can use it later if we ever want to force install a chart outside a project namespace (eg: CVMFS, although now cloudman-boot takes care of creating the namespace), while creating a project will always create a corresponding namespace (and error if the namespace exists already).